### PR TITLE
C#] Update after.md document for numbers exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: languages/csharp/exercises/concept/numbers/.docs/after.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# After
+
+One of the key aspects of working with numbers in C# is the distinction between integers (numbers with no digits behind the decimal separator) and floating-point numbers (numbers with zero or more digits behind the decimal separator).
```
Comment (robkeim) [inline languages/csharp/exercises/concept/numbers/.docs/after.md]: I've heard this terminology more commonly used... what do you think?
```suggestion
One of the key aspects of working with numbers in C# is the distinction between integers (numbers with no digits after the decimal separator) and floating-point numbers (numbers with zero or more digits after the decimal separator).
```

File: languages/csharp/exercises/concept/numbers/.docs/after.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Numbers can be compared using the default comparison operators (`<`, `>`, `==`, etc.)
+
+When converting between numeric types, there are two types of numeric conversions:
+
+1. Implicit conversions: no data will be lost and no additional syntax is required.
+2. Explicit conversions: data could be lost and additional syntax in the form of a _cast_ is required.
+
+As an `int` has less precision than a `double`, converting from an `int` to a `double` is safe and is thus an implicit conversion. However, converting from a `double` to an `int` could mean losing data, so that requires an explicit conversion.
+
+To conditionally execute logic, an `if` statement can be used.
```
Comment (robkeim) [inline languages/csharp/exercises/concept/numbers/.docs/after.md]: This sentence feels a bit out of place to me. Maybe there was something more you wanted to add? What kind of conditional logic would you want to execute related to numbers and casting?

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/numbers/.docs/after.md]: The logic has to deal with comparing the different speeds: https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/numbers/.meta/Example.cs#L16 I agree that it sounds weird. Any suggestions?

Comment (robkeim) [reply languages/csharp/exercises/concept/numbers/.docs/after.md]: How about if we move this phrase after the one currently on line 7 for something like:
Numbers can be compared using the default comparison operators (`<`, `>`, `==`, etc.). These comparisons can be used with `if` statements in order to conditionally execute code.

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/numbers/.docs/after.md]: Really like that! I've updated the PR.

**Comment by robkeim** [comment]
Thanks @ErikSchierboom!

**Pull request merged**